docs/schemas.md

```diff
@@ -176,11 +176,3 @@ The same applies when there is more than one level. The last child in the last l
 The other exception applies to JSON Schema: in order to support `null`values, the collections within objects (type : object) cannot be null. Therefore, they will be **required by default** and they will be **initialized as an empty collection** if the object than contains them is not null.
 
 > Within JSON Schema, a maximum of **2 nesting levels** is allowed.
-
-## Protobuf Schema
-
-### Not nested messages
-
-As the [Protobuf specification for nested types](https://developers.google.com/protocol-buffers/docs/proto3#nested) says, you can use message types inside other message 
-types, whether they are nested or not. KLoadGen supports it, but you have to define the messages in the same `.proto` file and add a `package` specifier to the `.proto` file on a 
-mandatory basis.
```

Comment:
Maybe you should explain how it will work with the new changes or what improvements have you added.

Reply:
It is not about any added functionality, or new casuistry. This is the correction of a bug in the processing of Protobuf Schemas.